### PR TITLE
Add i18n

### PR DESCRIPTION
**Pull request by mirhamasala**
- Add VueI18n library
- Configure locales

### CODE REVIEW

File: src/i18n/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Locales } from './locales';
+
+import en from './en.json';
+import jp from './jp.json';
```
Comment (yurikoval) [inline src/i18n/index.ts:4]: `jp` denotes a country, in this case it's a languages so it should be `ja`

File: src/i18n/locales.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export enum Locales {
+  EN = 'en',
+  JP = 'jp',
+}
+
+export const LOCALES = [
+  { value: Locales.EN, caption: 'English' },
+  { value: Locales.JP, caption: '日本人' },
```
Comment (yurikoval) [inline src/i18n/locales.ts:8]: I'm assuming this is for the language name - so it should be:
`日本人` -> `日本語 `

File: src/i18n/en.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "home": {
+    "headline": "The Ultimate Guide to COVID-19 Travel Restrictions"
```
Comment (yurikoval) [inline src/i18n/en.json:3]: How is this used?

Comment (mirhamasala) [reply src/i18n/en.json:3]: I didn't use it in any of the views yet if that's what you mean. I just set it up as a scaffold.

Or, if you mean how it's used _used_; It's like this: `<h1>{{ $t('home.headline') }}</h1>`

**Pull request merged**